include/aws/mqtt/private/client_impl.h

```diff
@@ -146,7 +150,7 @@ struct aws_mqtt_reconnect_task {
 /* The lifetime of this struct is from subscribe -> suback */
 struct subscribe_task_arg {
 
-    struct aws_mqtt_client_connection *connection;
```

Comment:
Will these changes, changing from `aws_mqtt_client_connection` to `aws_mqtt_client_connection_311_impl`, be a breaking change for anyone else using `aws-c-mqtt`? I don't know if anyone else outside our team is using `aws-c-mqtt` directly, but...

Reply:
Outside of that though, everything looks fine to me! I just want to take one more look through `source/client.c` as there was a couple areas I wasn't totally following when I read it initially.

Reply:
Weird, thought I replied.  Anyways, aws_mqtt_client_connection has always been opaque.  Internal changes to it are not visible to anyone using aws-c-mqtt directly.